cmd/generate.go

```diff
@@ -8,121 +8,20 @@ import (
 	"errors"
 	"fmt"
 	"net"
-	"os"
 	"strconv"
 	"strings"
 
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/srl-labs/containerlab/clab"
 	"github.com/srl-labs/containerlab/links"
-	"github.com/srl-labs/containerlab/nodes/ceos"
-	"github.com/srl-labs/containerlab/nodes/crpd"
-	"github.com/srl-labs/containerlab/nodes/rare"
-	"github.com/srl-labs/containerlab/nodes/sonic"
-	"github.com/srl-labs/containerlab/nodes/srl"
-	"github.com/srl-labs/containerlab/nodes/vr_sros"
-	"github.com/srl-labs/containerlab/nodes/vr_veos"
-	"github.com/srl-labs/containerlab/nodes/vr_vjunosevolved"
-	"github.com/srl-labs/containerlab/nodes/vr_vjunosswitch"
-	"github.com/srl-labs/containerlab/nodes/vr_vmx"
-	"github.com/srl-labs/containerlab/nodes/vr_vqfx"
-	"github.com/srl-labs/containerlab/nodes/vr_vsrx"
-	"github.com/srl-labs/containerlab/nodes/vr_xrv9k"
-	"github.com/srl-labs/containerlab/nodes/xrd"
+	"github.com/srl-labs/containerlab/nodes"
 	"github.com/srl-labs/containerlab/types"
+	"github.com/srl-labs/containerlab/utils"
 	"gopkg.in/yaml.v2"
 )
 
-func buildInterfaceFormatMap() map[string]string {
-	m := make(map[string]string)
-
-	// Add SR Linux kinds
-	for _, k := range srl.KindNames {
-		m[k] = "e1-%d"
-	}
-
-	// Add Arista cEOS kinds
-	for _, k := range ceos.KindNames {
-		m[k] = "eth%d"
-	}
```

Comment:
TODO: align interfaces names given some of them are using the "native" format, which was introduced after this PR existed.